css/area_footnotes.less

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file provides the design styles for the footnotes.
+ *
+ */
+
+/* + + + + + + + + + + + + + + + + + + + + + + + + + + */
+/* footnotes section  */
+.dokuwiki div.footnotes {
+  div.fn {
+    margin-bottom: .2rem;
+    display: table;//
+    vertical-align: top;//
+
+    sup, //
+    .content {
+      display: table-cell;
+    }
+
+    sup { //
+      vertical-align: top;
+
+      // footnote anchor
+      a.fn_bot {
+        font-size: .86em;
+        padding-right: .2em;
+      }
+    }
+
+    .content { //
+      vertical-align: top;
+      line-height: 135%;
+    }
+
+    > sup {
+
+    }
```

Comment:
Why is this empty? Shouldn't this be the only `<sup>` formatted as a table-cell? there might be regular `<sup>`s in the footnote content.

Reply:
good point, is done